python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -681,10 +681,13 @@ def _groupby_supported(gb):
     )
 
 
-def _make_name(*args, sep="_"):
-    """Combine elements of `args` into a new string"""
-    _args = (arg for arg in args if arg != "")
-    return sep.join(_args)
+def _make_name(col_name, sep="_"):
+    """Combine elements of `col_name` into a single string, or no-op if
+    `col_name` is already a string
+    """
+    if isinstance(col_name, tuple):
```

Comment:
If you reversed this, would it be more flexible? I don't know if `col_name` would ever be another iterable.
```
if isinstance(col_name, str):
    return col_name
return sep.join(name for name in col_name if name != "")
```